spec/models/playlist_spec.rb

```diff
@@ -21,12 +21,11 @@
     expect(playlist.tracks.include?(track)).to eq false
   end
 
-  it "sets another playlist to default if default is being destroyed" do
+  it "reset default to most recently updated playlist if default is destroyed" do
     radio  = FactoryBot.create :radio
-    playlist_1 = FactoryBot.create :playlist, radio: radio
-    playlist_2 = FactoryBot.create :playlist, radio: radio
+    playlist_1 = FactoryBot.create :playlist, radio: radio, updated_at: 2.days.ago
+    playlist_2 = FactoryBot.create :playlist, radio: radio, updated_at: 1.day.ago
```

Comment:
not strictly necessary for test to pass, but makes it clearer about why `playlist_2` is chosen as default rather than `playlist_1`

Reply:
I see, makes it a bit more explicit. :+1: